packages/virtualized-lists/Lists/VirtualizedList.js

```diff
@@ -616,22 +670,9 @@ class VirtualizedList extends StateSafePureComponent<Props, State> {
         ),
       };
     } else {
-      // If we have a positive non-zero initialScrollIndex and run this before we've scrolled,
-      // we'll wipe out the initialNumToRender rendered elements starting at initialScrollIndex.
-      // So let's wait until we've scrolled the view to the right place. And until then,
-      // we will trust the initialScrollIndex suggestion.
-
-      // Thus, we want to recalculate the windowed render limits if any of the following hold:
-      // - initialScrollIndex is undefined or is 0
-      // - initialScrollIndex > 0 AND scrolling is complete
-      // - initialScrollIndex > 0 AND the end of the list is visible (this handles the case
```

Comment:
Do you know why we need to recalculate the render limits with initialScrollIndex in the case where distanceFromEnd == 0?

With this change I no longer cover this case so I wonder if it is needed.

Reply:
I have wondered the same. It seems like its not really doing anything, so I think we can remove if everything still seems like its working.